### PR TITLE
Agenda Page model and template

### PR DESCRIPTION
**Pull request by johnnyporkchops**
Proposing the addition of a Wagtail model, AgendaPage and corresponding template: agenda_page.html. 
Example of current fec.gov page: http://www.fec.gov/agenda/2016/agenda20160915.shtml

### CODE REVIEW

File: fec/home/templates/home/agenda_page.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
+{% extends "base.html" %}
+{% block extra_css %}
+<style>
+ol.agenda_ol {
```
Comment (noahmanger) [inline fec/home/templates/home/agenda_page.html]: Let's make this a new class called `list--roman` and add it in [fec-style here](https://github.com/18F/fec-style/blob/reconcile-h1s/scss/components/_list-styles.scss)

File: fec/home/templates/home/agenda_page.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    list-style-type: upper-roman;
+}
+</style>
+{% endblock %}
+{% load wagtailcore_tags %}
+{% block body_class %}template-{{ self.get_verbose_name | slugify }}{% endblock %}
+
+{% block content %}
+<header class="page-header slab--secondary">
```
Comment (noahmanger) [inline fec/home/templates/home/agenda_page.html]: There's  new `breadcrumbs.html` partial that can be included here to generate breadcrumbs programmatically.

File: fec/home/templates/home/agenda_page.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+            </div>
+          {% endif %}
+        {% endfor %}
+      </aside>
+    </div>
+    {% endif %}
+  </div>
+</article>
+
+<div class="slab slab--secondary footer-disclaimer">
```
Comment (noahmanger) [inline fec/home/templates/home/agenda_page.html]: There's also a `disclaimer.html` partial that can be included here.

File: fec/home/templates/home/agenda_page.html (hunk 1 above)
Comment (johnnyporkchops) [inline fec/home/templates/home/agenda_page.html]: I have replaced the header block with:  {% include 'partials/breadcrumbs.html' with page=self links=self.get_ancestors style='secondary' %}

File: fec/home/templates/home/agenda_page.html (hunk 2 above)
Comment (johnnyporkchops) [inline fec/home/templates/home/agenda_page.html]: I have replaced the header block with:  {% include 'partials/breadcrumbs.html' with page=self links=self.get_ancestors style='secondary' %} and tested locally

File: fec/home/templates/home/agenda_page.html (hunk 3 above)
Comment (johnnyporkchops) [inline fec/home/templates/home/agenda_page.html]: I have replaced the footer block with: 
`{% include 'partials/disclaimer.html' %}`
And tested locally

File: fec/home/templates/home/agenda_page.html (hunk 1 above)
Comment (johnnyporkchops) [inline fec/home/templates/home/agenda_page.html]: For future reference, we using classic BEM and not the modified BEM that U.S. 
WDS is using?

Comment (noahmanger) [inline fec/home/templates/home/agenda_page.html]: Yes, classic BEM. Good question. I need to specify that in fec-style.

**Comment by johnnyporkchops** [comment]
The /media directory is most likely not correct and will need to be corrected to match the way we are uploading and managing static assets sitewide

**Comment by noahmanger** [comment]
We should get this up to date now that we're scraping the agendas. @johnnyporkchops can you take a pass at pulling in the latest from develop and sorting out merge conflicts? I can review and we can get it merged in.

Also cc @tadhg-ohiggins for visibility.

**Comment by johnnyporkchops** [comment]
Replaced by PR https://github.com/18F/fec-cms/pull/756

**Comment by johnnyporkchops** [comment]
Replaced by PR https://github.com/18F/fec-cms/pull/756

**Pull request closed without merging**